### PR TITLE
A few documentation corrections

### PR DESCRIPTION
**Pull request by risaacson**
Hey,

I was reading through the documentation and a few things started to jump out at me and I threw together fixes. One portion of the request is to clean up a few sentences that seemed a bit off. The rest if fixing a few "Note" blocks within the markup where the whole block was not showing or was improperly formatted.

I looked through the open tickets and I did not see anything that looks to be done to cover these issues.

tl;dr Easier to read and prettier documentation.

Thanks!

-r

### CODE REVIEW

**Comment by mpdehaan** [comment]
Merged, thank you!
